src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -176,6 +176,7 @@ def load_arguments(self, _):
         c.argument('load_balancer_outbound_ips', type=str, validator=validate_load_balancer_outbound_ips)
         c.argument('load_balancer_outbound_ip_prefixes', type=str, validator=validate_load_balancer_outbound_ip_prefixes)
         c.argument('vm_set_type', type=str, validator=validate_vm_set_type)
+        c.argument('zones', zones_type, options_list='--zones', help='Space-separated list of availability zones where agent nodes will be placed.')
```

Comment:
Maybe you can add an alias `-z` just like `vm/vmss create` to provide consistent experience across commands.

Reply:
Done